main.cpp

```diff
@@ -67,9 +90,22 @@ struct Wrapper
  Wait for my code review.
  */
 
+template<typename T>
+void variadicHelper(T&& first)
+{
+    Wrapper<T>(std::forward<T>(first)).print();
+}
+
+template<typename T, typename ...Args>
+void variadicHelper(T&& first, Args&& ... everythingElse)
+{
+    Wrapper<T>(std::forward<T>(first)).print();
+    variadicHelper(std::forward<Args>(everythingElse)...);
+}

```

Comment:
Project9.1 challenge 2:
There is another recursive solution.  it is possible for `Args ...` to be empty.
your task: remove your implementation for instruction `4)` and implement this alternate solution.
implement it **after** your existing `variadicHelper` function.
hint: what does the **recursive call** actually look like when called with an empty Variadic Parameter list?